opendata.swiss/ui/k8s/deployment.yaml

```
@@ -17,6 +17,12 @@ spec:
               value: https://piveau-hub-search-ln.zazukoians.org/
             - name: PIVEAU_HUB_REPO_URL
               value: https://piveau-hub-repo-ln.zazukoians.org/
+            - name: GITHUB_OWNER
+              value: opendata-swiss
+            - name: GITHUB_REPO
+              value: metadata.swiss
+            - name: GITHUB_BASE_BRANCH
+              value: main
```

I just configured the secret.
The environment variables could be configured that way:

```suggestion
              value: main
            - name: GITHUB_APP_ID
              valueFrom:
                secretKeyRef:
                  name: piveau-ui-github-app
                  key: app-id
            - name: GITHUB_APP_PRIVATE_KEY
              valueFrom:
                secretKeyRef:
                  name: piveau-ui-github-app
                  key: private-key
            - name: GITHUB_APP_INSTALLATION_ID
              valueFrom:
                secretKeyRef:
                  name: piveau-ui-github-app
                  key: installation-id
```